gitlab/gitlab-renew-cert

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#######################################################
+# File: gitlab-renew-cert                             #
+# Description: Renew RITlug GitLab Let's Encrypt cert #
+# Author: Tim Zabel <tjzabel@protonmail.com>          #
+#######################################################
+
+function renew_cert {
+
+    gitlab-ctl stop
```

Comment:
If you wanted to add better logging to a Bash script, you could use `logger` to send messages to the system journal.

See `man logger` for more info.